.github/workflows/release-pypi.yml

```diff
@@ -37,46 +47,86 @@ jobs:
     strategy:
       matrix:
         # Windows tests fail when built as a binary wheel for some reason
-        os: [ubuntu-latest, macos-latest]  #  windows-latest
+        # 'macos-12' doesn't pass the tests
+        os: [ubuntu-latest, macos-11] # windows-latest
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
 
     # setup Python for cibuildwheel
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v4
       with:
         python-version: '3.x'
 
     # for other architectures, see: https://cibuildwheel.readthedocs.io/en/stable/faq/#emulation
     - name: Build wheels for CPython
-      uses: pypa/cibuildwheel@v2.1.1
+      uses: pypa/cibuildwheel@v2.12.3
       env:
-        CIBW_BUILD: 'cp*'  # no PyPI builds
-        CIBW_SKIP: 'cp310-*'  # 3.10 builds are broken
-        CIBW_ARCHS: 'auto64'  # only 64-bit
-        CIBW_TEST_REQUIRES: pytest pytest-xdist
-        CIBW_TEST_COMMAND: pytest -n auto {project}/test/
-
-    - uses: actions/upload-artifact@v2
+        # cibuildwheel automatically reads 'python_requires' from 'setup.py'
+        CIBW_BUILD: 'cp*'  # no PyPy builds
+        CIBW_ARCHS_LINUX: "x86_64" # only Intel 64-bit
+        CIBW_ARCHS_MACOS: "x86_64" # only Intel 64-bit
+        # CIBW_TEST_REQUIRES: pytest pytest-xdist filelock 'setuptools>=60.2'
+        # CIBW_TEST_COMMAND: pytest --basetemp=.tmpdir -n auto {project}/test/
```

Comment:
What is the status of these commented out lines?

Reply:
I plan to re-enable this tests in this PR but ran out of time on Friday. The main problem is: we usually run tests _in-place_ but here, we really want to test the wheel. I'll finish that on Monday.